packages/react/src/components/Tab/Tab.js

```diff
@@ -121,7 +122,8 @@ export default class Tab extends React.Component {
       tabIndex,
```

Comment:
I don't think tabIndex should be a prop... you don't want authors messing with the tab order of the Tabs, because you want to control the switching when the user types arrow keys...

Reply:
it's mainly so that if the user chooses to set a custom tabindex, it is applied to the correct element rather than being spread onto the wrapper element. Typically the user isn't expected to provide a value for this since it's handled automatically by the component itself

Reply:
Regarding "custom tabindex":
- https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex
> Avoid using tabindex values greater than 0. Doing so makes it difficult for people who rely on assistive technology to navigate and operate page content.

- https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex
> Using a tabindex greater than 0 is considered an anti-pattern.

Reply:
in that case we can deprecate the prop to give consumers time to prepare for the prop removal